### PR TITLE
Update help info

### PR DESCRIPTION
**Pull request by Narimm**

### CODE REVIEW

**Review by addstar — APPROVED** [review]

File: src/main/java/me/botsko/prism/Prism.java [hunk 1]
```diff
@@ -332,7 +332,7 @@ public void onEnable() {
             monitor.addGauge(Prism.class, recordingQ, "RecordingQueueSize");
             ActionMeter.setupActionMeter(monitor.getRegistry());
         }
-        log("Initializing Prism " + plugin_version + ". By Viveleroi.");
+        log("Initializing Prism " + plugin_version + ". maintained my the AddstarMC Network");
```
Comment (viveleroi) [inline src/main/java/me/botsko/prism/Prism.java:335]: I think you meant `by`, not `my`.

File: src/main/java/me/botsko/prism/commands/AboutCommand.java [hunk 2]
```diff
@@ -28,12 +28,12 @@ public AboutCommand(Prism plugin) {
 	@Override
 	public void handle(CallInfo call) {
 		call.getSender().sendMessage(Prism.messenger.playerHeaderMsg(
-				"Prism - By " + ChatColor.GOLD + "viveleroi" + ChatColor.GRAY + " v" + plugin.getPrismVersion()));
```
Comment (viveleroi) [inline src/main/java/me/botsko/prism/commands/AboutCommand.java:31]: Is there any possibility to retain something that says, "originally by viveleroi" or something? Maybe on the next line?

File: src/main/java/me/botsko/prism/Prism.java (hunk 1 above)
Comment (addstar) [reply src/main/java/me/botsko/prism/Prism.java:335]: :+1: Good catch, thanks!

File: src/main/java/me/botsko/prism/commands/AboutCommand.java (hunk 2 above)
Comment (addstar) [reply src/main/java/me/botsko/prism/commands/AboutCommand.java:31]: Yep, we can do that :slightly_smiling_face: added in #146

Comment (Narimm) [reply src/main/java/me/botsko/prism/commands/AboutCommand.java:31]: I only really wanted to remove it so the people looking for support knew where to look

File: src/main/java/me/botsko/prism/Prism.java (hunk 1 above)
Comment (alchemistmatt) [reply src/main/java/me/botsko/prism/Prism.java:335]: Fixed via https://github.com/AddstarMC/Prism-Bukkit/pull/146

**Pull request merged**